client/my-sites/domains/domain-management/transfer/transfer-to-other-site/confirmation-dialog.jsx

```diff
@@ -49,26 +49,42 @@ class TransferConfirmationDialog extends PureComponent {
 	}
 
 	renderTargetSiteOnFreePlanWarnings() {
+		const { isMapping } = this.props;
+
 		if ( this.props.primaryWithPlansOnly ) {
-			return (
+			return isMapping ? (
 				<p>
 					{ this.props.translate(
 						"The target site doesn't have a paid plan, so you'll have to pay the full price for a " +
 							'domain mapping subscription when the domain mapping next renews. You will not be able to set it as primary either. ' +
 							'If you upgrade the target site to a paid plan, these features are included in the plan.'
 					) }
 				</p>
+			) : (
+				<p>
+					{ this.props.translate(
+						"The target site doesn't have a paid plan, so you'll have to pay the full price for a " +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'The target site doesn't have a paid plan, so you'll have to pay the full price for a domain mapping subscription when the domain mapping next renews. You will not be able to set it as primary either. If you upgrade the target site to a paid plan, these features are included in the plan.' )` *ES Score: 6*